deploy/examples/cluster.yaml

```diff
@@ -268,6 +268,19 @@ spec:
     #     deviceFilter: "^sd."
     # when onlyApplyOSDPlacement is false, will merge both placement.All() and placement.osd
     onlyApplyOSDPlacement: false
+    # OSD scrubbing schedule config, see Ceph Cluster CRD documentation section `OSD Scrubbing Schedule`
+    scrubbing:
+      # Whether to apply the scrubbing schedule given here to the cluster
+      enabled: false
```

Comment:
scrubbing enabled false, should be highly discouraged, it should default to true as well

Reply:
I hoped that the comment would describe it well enough that this only means if the "scrubbing schedule is applied" to the cluster.

Should I change the line comment to something else or do you have other ideas for a "toggle for the scrubbing schedule settings" specified in that structure?